### DISCUSSION TITLE
Calling multiple devices at the same time will lose the connection

### DISCUSSION THREAD

**Discussion by MikasaGitHub** [IchthysMaranatha/asterisk-chan-quectel#58] [Q&A] [answered]
Hello, I currently have two devices, quectel0 and quectel1. When I use one device to make a call, the other party can connect and talk. I keep the first call alive. If I use the second device to call another person at this time, the two devices will lose connection.
When using quectel show devices, both devices cannot be found. I don’t know why. Looking forward to your reply.

**Comment by IchthysMaranatha** [comment, marked as answer]
I have not tested with multiple devices, but there have been others who have used multiple devices without this particular problem in the discussions. My guess would be a power issue, your equipment may not be able to provide power to both devices operating together.

**Reply by MikasaGitHub** [reply]
Thanks for your reply, I'm using a Raspberry Pi. I will verify this reason later.
